node/src/p2p.rs

```diff
@@ -368,12 +414,19 @@ where
             header_sub_topic_hash: header_sub_topic.hash(),
             peer_tracker,
             header_sub_watcher,
-            store: args.store.clone(),
         })
     }
 
     async fn run(&mut self) {
+        let mut interval = Interval::new(Duration::from_secs(60)).await;
+        let mut first_report_showed = false;
```

Comment:
`first_report_shown` I think? Similar to the other interval, `report_log_interval`?